typings/index.d.ts

```diff
@@ -2291,7 +2297,7 @@ declare module 'discord.js' {
 	interface GuildCreateChannelOptions {
 		permissionOverwrites?: OverwriteResolvable[] | Collection<Snowflake, OverwriteResolvable>;
 		topic?: string;
-		type?: Exclude<keyof typeof ChannelType, 'dm' | 'group' | 'unknown'>;
+		type?: Exclude<ChannelType, ChannelType.DM | ChannelType.GROUP | ChannelType.UNKNOWN>;
```

Comment:
Won't this change prevent using numbers for the type? Did my old change cause that too?

Reply:
That's a good point Vlad, it will.

Not sure what to do about this, honestly.

Reply:
I mean.. won't reverting the enum change itself fix it? By default, enum entries start at 0, which is also why I made the keys lowercased (to match what djs can parse)

Reply:
But then that defeats the whole purpose of this PR, of not being forced to use string literals for the individual `type` properties

Reply:
You would still be forced to use either string literals or numbers for type guarding the channels.

Reply:
Type guarding yeah, I'm talking about the literal typings